docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -88,38 +88,6 @@ Additional information about using this image is available in the
 [Oracle Container Registry](https://container-registry.oracle.com).
 
 
-#### Obtaining the FMW Infrastructure Docker image
```

Comment:
Is this duplicated text?  I don't understand why you are deleting this content... :)

Reply:
The "Obtaining the FMW Infrastructure Docker image" section was duplicated in my branch. However, now when I look in Master, I'm not seeing it that way. Do you?

Reply:
That sounds like a good idea. Perhaps, you can create a JIRA for it and assign it to me. I'd be glad to do the work when I am able.

Reply:
Okay, yes, I also see it duplicated.